templates/base.html

```diff
@@ -32,7 +32,15 @@
                     <ul class="nav navbar-nav">
                     </ul>
                     <ul class="nav navbar-nav navbar-right">
-                        <li><a href="{% url 'users:signup' %}">Sign up</a></li>
+						{% if user %}
+							<li><a href="{% url 'users:profile' pk=user.pk %}">@{{ user.username }}</a></li>
```

Comment:
f7bdf7eeadbd7e329b51ec8e4bee3ca457a41c67 fixes this, specifically [this line](https://github.com/7Pros/circuit/commit/f7bdf7eeadbd7e329b51ec8e4bee3ca457a41c67#diff-8944b57466f08564caa53a1988261ae0L35).
You might revert 1a3d8d214b6b6b361dd41c31285f0dfae07a3101 and patch it this way.